R/get.R

```diff
@@ -508,7 +508,7 @@ get_args <- function(formal.args, dot.args) {
 #' get_param_set(mod)
 #' @export
 get_param_set <- function(sims) {
-  if (class(sims) != "netsim") {
+  if (!any(class(sims) %in% "netsim")) {
```

Comment:
another function for this is: `inherits(sims, "netsim")`

Reply:
I understood where was the problem with the object having multiple classes. 
With a `netsim` object it works perfectly as it is only a "netsim". However, in the tests I try using `get_param_set` on a "control" object. I get the error as expected but also a warning because the `control` object is of class `c("control.net", "control.list", "list")`and the `if` statement only checks the first one. 

Using `inherits` as you suggest will solve that and make the code work as expected each time